quick-start/src/main/ui/app/flows/flows.component.ts

```diff
@@ -1,35 +1,22 @@
-import { Component, EventEmitter, OnInit, OnDestroy, QueryList, ViewChildren, ViewChild } from '@angular/core';
+import { MdlDialogService, MdlSnackbarService } from '@angular-mdl/core';
+import { Component, EventEmitter, OnDestroy, OnInit, QueryList, ViewChild, ViewChildren, ChangeDetectionStrategy } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
+import { distanceInWords } from 'date-fns';
+import * as _ from 'lodash';
+import { Observable } from 'rxjs/Rx';
 
+import { DeployService } from '../deploy/deploy.service';
+import { EntitiesService } from '../entities/entities.service';
 import { Entity } from '../entities/entity.model';
 import { Flow } from '../entities/flow.model';
 import { Plugin } from '../entities/plugin.model';
-
-import { EntitiesService } from '../entities/entities.service';
-
-import { MdlSnackbarService } from '@angular-mdl/core';
-
-import { MdlDialogService, MdlDialogReference } from '@angular-mdl/core';
-
-import { MlcpUiComponent } from '../mlcp-ui';
+import { EnvironmentService } from '../environment';
 import { HarmonizeFlowOptionsComponent } from '../harmonize-flow-options/harmonize-flow-options.component';
-import { NewEntityComponent } from '../shared/components/new-entity/new-entity.component';
-import { NewFlowComponent } from '../new-flow/new-flow.component';
-
 import { JobListenerService } from '../jobs/job-listener.service';
-import { EnvironmentService } from '../environment';
-
+import { NewFlowComponent } from '../new-flow/new-flow.component';
 import { HasBugsDialogComponent } from '../shared/components';
-
-import { DeployService } from '../deploy/deploy.service';
-
 import { CodemirrorComponent } from '../shared/components/codemirror';
-
-import { Ng2DeviceService } from 'ng2-device-detector';
-
-import { distanceInWords } from 'date-fns';
-
-import * as _ from 'lodash';
+import { NewEntityComponent } from '../shared/components/new-entity/new-entity.component';
 
 @Component({
   selector: 'app-flows',
```

Comment:
html file is excessive in smart component and template option should be used instead... also scss file must be in ui component location and should be deleted from here.